hongit-api/src/main/kotlin/com/devfloor/hongit/api/article/presentation/ArticleController.kt

```diff
@@ -34,13 +36,38 @@ class ArticleController(
 
     @GetMapping(params = ["boardId"])
     @ResponseStatus(value = HttpStatus.OK)
-    fun showAllByBoardId(@RequestParam boardId: Long): List<ArticleFeedResponse> =
-        articleService.showAllByBoardId(boardId)
+    fun showAllByBoardId(
+        @RequestParam boardId: Long,
+        @RequestParam(required = false) sort: ArticleSortType?,
+        @RequestParam(required = false) options: List<Long>?,
+        @RequestParam page: Int,
+        @RequestParam pageSize: Int,
+    ): List<ArticleFeedResponse> =
+        articleService.showAllByBoardId(boardId, page, pageSize, sort, options)
+            .also { log.info("boardId = $boardId, sort = $sort, options = $options") }
```

Comment:
```suggestion
    fun showAllByBoardId(
        @RequestParam boardId: Long,
        @RequestParam page: Int,
        @RequestParam pageSize: Int,
        @RequestParam(required = false) sort: ArticleSortType?,
        @RequestParam(required = false) options: List<Long>?,
    ): List<ArticleFeedResponse> =
        articleService.showAllByBoardId(boardId, page, pageSize, sort, options)
            .also { log.info("boardId = $boardId, sort = $sort, options = $options") }
```

이것도 아래 서비스의 메서드랑 한번에 이해하기 쉽게 순서 맞춰주기만 해도 좀 더 좋을 것 같아서 추천해봄요

Reply:
반영완료